es/index.js

```diff
@@ -102,7 +108,7 @@ export default function axiosBunyan(axios, {
 }) {
   const logRequest = (config) => {
     // eslint-disable-next-line no-param-reassign
-    config.timestamp = new Date().toISOString();
+    config[namespace] = Object.assign({ timestamp: Date.now() }, config[namespace]);
```

Comment:
**This is the relevant change**. If the config already contains a `timestamp` we keep it, so when when log the response time it starts counting from the very first request.